### PR TITLE
User Configureable Styles

### PR DESCRIPTION
**Pull request by aaronmarkey**
Thought it would be cool to allow users to easily update some simple styling elements of hyper-search to better fit their Hyper setup. 

* Allows users to modify the border radius of buttons, and the input field.
* Allows users to modify margin between buttons.
* Allows users to change prev/next button text (undocumented in README.)

### CODE REVIEW

File: src/containerStyles.js [hunk 1]
```diff
@@ -1,75 +1,100 @@
...
-    width: '12%',
-    borderRadius: '12px 12px 12px 12px',
-  }
-};
+exports.decorateConfig = (config) => {
+  const hyperSearchUI = Object.assign({
+    inputBorderRadius: 10,
+    buttonBorderRadius: 10,
+    prevButton: '◀',
+    nextButton: '▶',
```
Comment (jaanauati) [inline src/containerStyles.js:24]: could u please add a little description about these two to the readme file? (prev and nextButton, )

File: README.md [hunk 2]
```diff
@@ -17,6 +17,60 @@ Usage:
...
+      hyperSearchUI: {
+        buttonMargin: 2,
+      }
+    ...
+  }
+}
+````
+
+![button margin](https://i.imgur.com/ZKk0mO8.png)
+
```
Comment (jaanauati) [inline README.md:73]: Can u please add your name to the Contributors section? (this same file, line 81, names are listed alphabetically).

**Review by jaanauati — CHANGES_REQUESTED** [review]

**Comment by jaanauati** [comment]
This looks good!, I tested the changes in my local and everything works well, regarding my two requests, I'd like you to make the tweaks so the screenshots use the same styles (I have a different styling setup), but please, let me know in case that u don't have time (I could merge this and deal w the docs later).

Thank you!

**Comment by aaronmarkey** [comment]
Updated the README with your requests. I wasn't able to accurately replicate your setup locally, so the screens are still using mine.

**Comment by jaanauati** [comment]
Your screenshots are perfect, hyper-search@0.2.0 has been pushed to npm.

Thank you!

**Pull request merged**